packages/mdc-tab-scroller/_mixins.scss

```diff
@@ -35,6 +57,8 @@
     @include mdc-feature-targets($feat-structure) {
       overflow-y: hidden;
     }
+
+    @include mdc-tab-scroller-transition($mdc-tab-scroller-transition-duration, $query)
```

Comment:
Note this is a change in the generated CSS. Before:

```css
.mdc-tab-scroller--animating .mdc-tab-scroller__scroll-content {...}
```

whereas now it will be:

```css
.mdc-tab-scroller .mdc-tab-scroller--animating .mdc-tab-scroller__scroll-content {...}
```

The added specificity shouldn't be a problem since MDC doesn't support style overrides except via mixin/variable, correct?